docs/intro.md

```diff
@@ -9,13 +9,13 @@
 The intent of `Ard` is to be a modular, full-stack multi-disciplinary optimization tool for wind farms.
 
 The problem with wind farms is that they are complicated, multi-disciplinary objects.
```

Comment:
same comments as above, and we should probably not be duplicating content. Can we organize such that we don't repeat our content in more than one place?